doc/nrf/ug_nrf5340.rst

```diff
@@ -356,107 +356,112 @@ You must manually add a network core project to the application core project to
 
-       Adding script files for programming the network core
+         Adding script files for programming the network core
 
-#. Click :guilabel:`Next`.
+   #. Click :guilabel:`Next`.
 
-#. Add the project configurations.
-   This project will only be used for programming the network core so no build configurations are needed.
+   #. Add the project configurations.
+      This project will only be used for programming the network core, so no build configurations are needed.
 
-   Deselect :guilabel:`Debug` and :guilabel:`Release`.
+      Deselect :guilabel:`Debug` and :guilabel:`Release`.
 
-    .. figure:: images/ses_nrf5340_netcore_conf.png
-       :alt: Deselecting configurations and finishing the configuration
+      .. figure:: images/ses_nrf5340_netcore_conf.png
+         :alt: Deselecting configurations and finishing the configuration
 
-       Deselecting configurations and finishing the configuration
+         Deselecting configurations and finishing the configuration
 
-#. Click :guilabel:`Finish`.
+   #. Click :guilabel:`Finish`.
 
-   This creates a new project for programming the network core with the HEX file of the network sample.
+      This creates a new project for programming the network core with the HEX file of the network sample.
 
 #. Set the new network core project as the active project using :guilabel:`Project` > :guilabel:`Set Active Project`.
    For example, select :guilabel:`hci_rpmsg_nrf5340_netcore`.
 
    .. figure:: images/ses_nrf5340_netcore_set_active.png
-     :alt: Set the hci_rpmsg_nrf5340_netcore programming target as active
+      :alt: Set the hci_rpmsg_nrf5340_netcore programming target as active
 
-     Set the hci_rpmsg_nrf5340_netcore programming target as active
+      Set the hci_rpmsg_nrf5340_netcore programming target as active
 
 #. Program the network sample using :guilabel:`Target` > :guilabel:`Download XXX` (for example, :guilabel:`Download hci_rpmsg_nrf5340_netcore`).
 
    .. figure:: images/ses_nrf5340_netcore_flash_active.png
-     :alt: Program the network sample hci_rpmsg_nrf5340_netcore
+      :alt: Program the network sample hci_rpmsg_nrf5340_netcore
 
-     Program the network sample hci_rpmsg_nrf5340_netcore
+      Program the network sample hci_rpmsg_nrf5340_netcore
 
-   Ignore any warnings regarding the project being out of date.
+   Ignore any warnings regarding the project being out of date by clicking :guilabel:`No` when they appear.
    The network core project is a pure programming target, so it cannot be built.
 
    .. figure:: images/ses_nrf5340_netcore_download.png
-     :alt: Ignore any 'Project out of date' warning
+      :alt: Ignore any 'Project out of date' warning
 
-     Ignore any 'Project out of date' warning
+      Ignore any 'Project out of date' warning
+
+   .. warning::
+      If you click :guilabel:`Yes` and untick the option to show the dialog again, you will enter a loop because of a "no input files" error.
```

Comment:
```suggestion
      If you click :guilabel:`Yes` and disable the option to show the dialog again, you will enter a loop because of a "no input files" error.
```